synthtool/gcp/templates/python_library/.kokoro/release.sh

```diff
@@ -16,7 +16,7 @@
 set -eo pipefail
 
 # Start the releasetool reporter
-python3 -m pip install --require-hashes -r .kokoro/requirements.txt
+python3 -m pip install --require-hashes -r github/{{ metadata['repo']['repo'].split('/')[1] }}/.kokoro/requirements.txt
```

Comment:
Should it be `metadata['repo'].split('/')[1]`?

Reply:
It looks strange but I tested the changes in https://github.com/googleapis/python-securitycenter/pull/378 and the release script succeeded.

I copied the path from the following line in the same file
https://github.com/googleapis/synthtool/blob/037c0a98c6541954e73384a9089a38f1294adfa0/synthtool/gcp/templates/python_library/.kokoro/release.sh#L27

Reply:
Thanks for confirming!